src/test/groovy/org/osiam/test/integration/AbstractIT.groovy

```diff
@@ -42,32 +42,45 @@ import javax.sql.DataSource
  */
 abstract class AbstractIT extends Specification {
 
-    protected static final String CLIENT_ID = "example-client"
-    private static final String CLIENT_SECRET = "secret"
+    private static final String CLIENT_ID = 'example-client'
+    private static final String CLIENT_SECRET = 'secret'
 
-    protected static final String AUTH_ENDPOINT = "http://localhost:8180/osiam-auth-server"
-    protected static final String RESOURCE_ENDPOINT = "http://localhost:8180/osiam-resource-server"
-    protected static final String REGISTRATION_ENDPOINT = "http://localhost:8180/addon-self-administration"
+    private static final String AUTH_ENDPOINT = 'http://localhost:8180/osiam-auth-server'
+    protected static final String RESOURCE_ENDPOINT = 'http://localhost:8180/osiam-resource-server'
+    protected static final String REGISTRATION_ENDPOINT = 'http://localhost:8180/addon-self-administration'
 
     protected static final String SELF_ADMIN_URN = 'urn:org.osiam:scim:extensions:addon-self-administration'
 
-    protected OsiamConnector osiamConnector
-    protected OsiamConnector osiamConnectorForClientCredentialsGrant
-    protected OsiamConnector osiamConnectorForEmailChange
+    protected static OsiamConnector OSIAM_CONNECTOR
+    protected static OsiamConnector OSIAM_CONNECTOR_FOR_CLIENT_CREDENTIALS
+
+    protected AccessToken accessToken
 
     static {
         OsiamConnector.setConnectTimeout(10000);
         OsiamConnector.setReadTimeout(10000);
-    }
 
-    def AccessToken accessToken
+        OSIAM_CONNECTOR = new OsiamConnector.Builder()
+                .setAuthServerEndpoint(AUTH_ENDPOINT)
+                .setResourceServerEndpoint(RESOURCE_ENDPOINT)
+                .setClientId(CLIENT_ID)
+                .setClientSecret(CLIENT_SECRET)
+                .build()
+
+        OSIAM_CONNECTOR_FOR_CLIENT_CREDENTIALS = new OsiamConnector.Builder()
+                .setAuthServerEndpoint(AUTH_ENDPOINT)
+                .setResourceServerEndpoint(RESOURCE_ENDPOINT)
+                .setClientId(CLIENT_ID)
+                .setClientSecret(CLIENT_SECRET)
+                .build()
```

Comment:
These are both the same connectors. I think this is a legacy from a time when we did not support distinct OAuth flows using the same connector instance. as this is now possible, we should only create 1 connector.